src/bp/migrations/v12_10_0-1591119240-migrate_content_in_flow_data_for_ndu_bots.ts

```diff
@@ -34,10 +34,12 @@ async function migrateFlow(
     }
 
     const element = contentMap[contentId]
-    n.content = {
-      contentType: contentId.substring(0, contentId.lastIndexOf('-')),
-      formData: element.formData
-    }
+    n.contents = [
+      {
+        contentType: contentId.substring(0, contentId.lastIndexOf('-')),
+        formData: element.formData
+      }
+    ]
     n.onEnter = []
```

Comment:
You cannot edit an existing migration, especially if the structure changes. This migration will not run for users who are already migrated on 12.10. You will need to create another one for the future version,.

Reply:
yeah that's what I was thinking, but I couldn't build because it wasn't respecting the typing changed for contents... do I need to just leave content and contents to the typing of the say node?